server.go

```diff
@@ -2048,17 +2048,37 @@ func (s *server) peerTerminationWatcher(p *peer) {
 // nextPeerBackoff computes the next backoff duration for a peer's pubkey using
 // exponential backoff. If no previous backoff was known, the default is
 // returned.
-func (s *server) nextPeerBackoff(pubStr string) time.Duration {
+func (s *server) nextPeerBackoff(pubStr string,
+	startTime time.Time) time.Duration {
+
 	// Now, determine the appropriate backoff to use for the retry.
 	backoff, ok := s.persistentPeersBackoff[pubStr]
 	if !ok {
 		// If an existing backoff was unknown, use the default.
 		return defaultBackoff
 	}
 
-	// Otherwise, use a previous backoff to compute the
-	// subsequent randomized exponential backoff duration.
-	return computeNextBackoff(backoff)
+	// If the peer failed to start properly, we'll just use the previous
+	// backoff to compute the subsequent randomized exponential backoff
+	// duration. This will roughly double on average.
+	if startTime == lnpeer.StartTimeFailed {
```

Comment:
Can just use `IsZero()`. I think that makes more sense since we are not really setting the start time equal to `StartTimeFailed` anywhere.

Reply:
Fixed!